iis-wf/iis-wf-referenceextraction/src/main/resources/eu/dnetlib/iis/wf/referenceextraction/dataset/output_transformer/oozie_app/workflow.xml

```diff
@@ -75,7 +75,7 @@
             <main-class>eu.dnetlib.iis.common.java.ProcessWrapper</main-class>
             <arg>eu.dnetlib.iis.common.report.PigCountersReportGenerator</arg>
             <arg>-PpigCounters=${hadoop:counters('transformer')}</arg>
-            <arg>-Preport.processing.referenceExtraction.dataset.total.reference={dedupDocumentToDataset.RECORD_WRITTEN}</arg>
```

Comment:
I think due to massive changes in [mapping spreadsheet](https://docs.google.com/spreadsheets/d/1q3Kx1zwW3zrSWvfeI6ePM4i3338SymOvLN86z8Xap90/edit#gid=712041498) we have overlooked this case as a candidate for `total`->`deduped` rename. I guess we could align it with `processing.citationMatching.*` and `processing.docOrgMatching.*` cases.

Especially seeing the source for this metric defined as `dedupDocumentToDataset` :).

Reply:
One argument for this change: avoiding enquiries about `total` not being equal `datacite`+`opentrials` :) 

This would somehow let us use `total` metric name suffix ONLY for the 3 "aggregated" cases, namely: 
* `processing.referenceExtraction.community.references`
* `processing.referenceExtraction.concept.references`
* `processing.referenceExtraction.project.references`

where sum of all `byrootid.*` / `byfunder.*` metrics is in fact equal to its `total` metric.

Reply:
Ok, renaming `total` -> `deduped`, it was a good decision to split the renaming into separate PR, because I think we could have missed things like this :)